src/app/components/autocomplete/autocomplete.spec.ts

```diff
@@ -163,8 +163,8 @@ describe('AutoComplete', () => {
         inputEl.nativeElement.click();
         fixture.detectChanges();
 
-        const onKeydownSpy = spyOn(autocomplete, 'onKeydown').and.callThrough();
-        const onKeyupSpy = spyOn(autocomplete, 'onKeyup').and.callThrough();
+        const onKeydownSpy = spyOn(autocomplete, 'onKeyDown').and.callThrough();
+        const onKeyupSpy = spyOn(autocomplete, 'onKeyUp').and.callThrough();
```

Comment:
Here should be `onKeyUpSpy` and `onKeyDownSpy`

Reply:
ya, I miss those spelling sorry for that next time.i will be more careful.

Reply:
> ya, I miss those spelling sorry for that next time.i will be more careful.

No problem bro, thanks for your contribution :)